packaging/systemd/cloudstack-management.service

```diff
@@ -33,7 +33,7 @@ Type=simple
 User=cloud
 EnvironmentFile=/etc/default/cloudstack-management
 WorkingDirectory=/var/log/cloudstack/management
-PIDFile=/var/run/cloudstack-management.pid
+PIDFile=/run/cloudstack-management.pid
```

Comment:
This will likely fail on CentOS7, and older Ubuntu 16.04/18.04

Reply:
20.04 is a mess, this is like they changed lot of things as they did with 18.04 when they suddenly removed `ifconfig`